src/screens/home/components/tokenomics/index.tsx

```diff
@@ -12,7 +12,11 @@ import {
   Cell,
```

Comment:
This is not tokenomics component anymore, but validatorSetChart or something like that, let's create new component and leave tokenomics unchanged, or maybe delete it (in that case renaming would work as well).

Also, component title should be Validators instead of Tokenomics, and we don't need stake info below title since it is visible on other component next to it (Online voting power).

Reply:
Alright, makes sense - i will do that 👍🏽